cypress/page/address-step.page.ts

```diff
@@ -2,11 +2,11 @@ class AddressStepPage {
   private address: string;
 
   constructor() {
-    this.address = ".cart_navigation.clearfix > button";
+    this.address = "Proceed to checkout";
   }
 
   public goToCheckout(): void {
-    cy.get(this.address).click();
+    cy.get("p").contains(this.address).click();
```

Comment:
if the contains does not work alone, is better to use a CSS selector using an attribute that contains the text "Proceed to checkout"